security/pkg/nodeagent/sds/sdsservice_test.go

```diff
@@ -110,6 +154,17 @@ func TestStreamSecretsForGatewaySds(t *testing.T) {
 	testHelper(t, arg, sdsRequestStream, false)
 }
 
+func TestStreamSecretsForLocalBothSds(t *testing.T) {
```

Comment:
I don't think this test is needed. It is the same as TestStreamSecretsForBothSds(), and gateway SDS is not using token.

Reply:
remove